index.bs

```diff
@@ -3555,13 +3558,27 @@ This [=registration extension=] and [=authentication extension=] allows for a si
 : Authenticator extension input
 :: The client extension input encoded as a CBOR text string (major type 3).
 
+<pre>
+CDDL:
+(
+txAuthSimple: tstr,
+)
+</pre>
+
 : Authenticator extension processing
 :: The authenticator MUST display the prompt to the user before performing either [=user verification=] or [=test of user
     presence=]. The authenticator may insert line breaks if needed.
 
 : Authenticator extension output
 :: A single CBOR string, representing the prompt as displayed (including any eventual line breaks).
 
+<pre>
+CDDL:
+(
+txAuthSimple: tstr,
+)
+</pre>
+
```

Comment:
I'm not sure we need this CDDL representation of the extension output. If we were to use something like it, we'd want to go thru and apply the same treatment to all extension definitions IMO.

also I think the CDDL is not quite what we'd want, it'd be more like..
```
txAuthSimpleRetVal = ( tstr )  
```
...I think (?)